WordPress/build.gradle

```diff
@@ -156,8 +156,8 @@ configurations.all {
 }
 
 task generateCrashlyticsConfig(group: "generate", description: "Generate Crashlytics config") {
-    def outputFile = new File("${rootDir}/WordPress/crashlytics.properties")
-    def inputFile = file("${rootDir}/WordPress/gradle.properties")
+    def outputFile = new File("${rootDir}/crashlytics.properties")
```

Comment:
This won't work Crashlytics needs his properties in the app root (ie `WordPress/`)